kwanj-k/store_manager
# #161792599 Cart update

### What does this PR do?
Consumes the update, delete and checkout cart endpoints

#### Description of Task to be Completed ?
* A user should be able to update or remove a product from a cart
* A user should be able to check out a cart or delete an entire cart

### How should this be manually tested?
After cloning the repo, git pull the branch ft-cart-update-#161792599
* Then open the index.html file.
    ```
      Create a store and log in.
      Add a product and add it to cart.
      Click on the cart to see the pending cart.
      Try update, delete or checkout buttons

    ```

### What are the relevant Pivotal Tracker stories?
#161792599
